rust/libnewsboat/src/utils.rs

```diff
@@ -94,5 +102,17 @@ mod tests {
         assert_eq!(to_u(String::from(""), 0), 0);
         assert_eq!(to_u(String::from("zero"), 1), 1);
     }
+
+    #[test]
+    fn t_get_default_browser() {
+        use std::env;
+        let key = "BROWSER";
+        env::remove_var(key);
```

Comment:
This call modifies environment for *all* tests, which might hide bugs or introduce failures. Please split it into a file under `tests` directory (it contains integration tests, which are run completely separately from each other).

(In C++, we have a separate class that uses RAII to restore the variable when the test finishes running. I think it could be ported for uses like that, but it's not a high-priority task to me. I think it's fine to use integration tests for this.)

Reply:
okay i will work on it